Core/Core/Pages/PageDetails/PageDetailsViewController.swift

```diff
@@ -86,7 +94,7 @@ public final class PageDetailsViewController: UIViewController, ColoredNavViewPr
         } else {
             groups.refresh()
         }
-        pages.refresh(force: true)
+        pages.refresh(force: false)
```

Comment:
Is it safe to turn this to false? Usually, these are added to work around a bug.